homeassistant/components/mqtt/abbreviations.py

```diff
@@ -83,6 +83,7 @@
     "hs_cmd_tpl": "hs_command_template",
     "hs_stat_t": "hs_state_topic",
     "hs_val_tpl": "hs_value_template",
+    "i": "integration",
```

Comment:
Let's change this to `origin` instead to not confuse it with Home Assistant integrations.